inference-engine/samples/hello_nv12_input_classification/main.cpp

```diff
@@ -227,10 +212,18 @@ int main(int argc, char *argv[]) {
         CNNNetwork network = ie.ReadNetwork(input_model);
         // -----------------------------------------------------------------------------------------------------
 
-        // --------------------------- 2. Set model batch size -------------------------------------------------
+        // --------------------------- 2. Reshape model -------------------------------------------------
         size_t batch_size = isBatchedBlobSupported(ie, device_name) ? image_names.size() : 1;
         std::cout << "Setting network batch size to " << batch_size << std::endl;
-        setBatchSize(network, batch_size);
+        ICNNNetwork::InputShapes inputShapes = network.getInputShapes();
+        for (auto& shape : inputShapes) {
+            auto& dims = shape.second;
+            if (dims.empty()) {
+                throw std::runtime_error("Network's input shapes have empty dimensions");
+            }
+            dims[0] = batch_size;
```

Comment:
I think we can use another naming here instead of batch_size,  e.g. netInputSize?